recipes/libdispatch/all/conanfile.py

```diff
@@ -0,0 +1,64 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.32.0"
+
+class LibDispatchConan(ConanFile):
+    name = "libdispatch"
```

Comment:
There's a lot of forks of this... perhaps an `apple` prefix needs to be added

Reply:
Sorry, forks of what?
libdispatch?

Reply:
It is from Apple, you are right, but I think the one from Apple is the original one.

https://github.com/apple/swift-corelibs-libdispatch

Reply:
> forks of what? libdispatch?

Yes

I think it was OSS that apple ran with since they needed it

Reply:
What I know of is a project run by Apple.
Any variant is a fork of Apple's.